source/tests/system/nidaqmx_driver_api_tests.cpp

```diff
@@ -675,12 +675,12 @@ class NiDAQmxDriverApiTests : public Test {
     return stub()->CfgWatchdogDOExpirStates(&context, request, &response);
   }
 
-  void raise_if_error(::grpc::Status status)
-  {
-    if (!status.ok()) {
-      throw new std::runtime_error(status.error_message());
-    }
-  }
+  // void raise_if_error(::grpc::Status status)
```

Comment:
Should we just delete this commented out `raise_if_error` function?

Reply:
Thanks, I knew I forgot something. :)